### DISCUSSION TITLE
can not see the relationship between actors and parent in stately.ai/viz

### DISCUSSION THREAD

**Discussion by OnlySilva** [statelyai/xstate#3123] [Q&A]
**Description**
As a newcomer to xstate,I want to see the relationship between actors and parent,like 
<img width="431" alt="image" src="https://user-images.githubusercontent.com/6448800/156907332-ec3a0e1b-b0dc-4b55-b140-a7892592dd97.png">

However,I can only get the parent machine statechart,just like this
https://stately.ai/viz/083738b7-56c2-4eb8-8ac3-c57dce74862c

I have read the feature:
https://github.com/statelyai/xstate/issues/428, 
but still be confused about how to get the picture above
Thank you.

**Comment by davidkpiano** [comment]
This is something we've been thinking about, and still need to explore. It's on our todo list!
